commands/init.js

```diff
@@ -299,3 +298,11 @@ function resetValues () {
   target = null;
   typeScript = null;
 }
+
+function isComponentsAppType (answers) {
+  return answers.APPTYPE === 'Components';
+}
+
+module.exports.testIsComponentsAppType = function (answers) {
```

Comment:
I'm not really happy with you exporting this just for tests.  It makes more sense to move this function to UTIL and then it will be exported there.

Reply:
Done 👍